app/templates/check_email.html

```diff
@@ -53,7 +53,7 @@ <h5 class="card-title">{% trans %}Check configuration by sending an e-mail{% end
                                 </div>
                             </div>
 
-                            <img class="spinner waiting" src="/static/images/spinner.svg" />
+                            <img class="spinner waiting" src="/static/images/spinner.svg" alt="spinner-waiting-img"/>
```

Comment:
As 'alt' will be read by screen reader, would it be possible to add a human-readable message? E.g. "Waiting" - the Polish translation would be "Trwa oczekiwanie".

Reply:
Hi,
So far I can't seem to find a way to add a translation to an "alt" attribute without modifying the {{% trans % }} or making significant changes that would seem out of touch of your current translations and just changing the logic of the translations since I want to keep it the same way as it is. I will look further into this next week. We could just change the alt to "Waiting" and keep it like that as a general message without translation but ofcourse the translation would look prettier and more refined.

Reply:
I added something but please check whether the Lithuanian translation is OK, I don't know Lithuanian ;)